packages/tables/resources/views/components/columns/layout.blade.php

```diff
@@ -6,7 +6,7 @@
 ])
 
 @php
-    $getHiddenClasses = function (\Filament\Tables\Columns\Column | \Filament\Tables\Columns\Layout\Component $layoutComponent): ?string {
+    $getHiddenClasses = function (Filament\Tables\Columns\Column | Filament\Tables\Columns\Layout\Component $layoutComponent): ?string {
```

Comment:
Might have to check this

Reply:
@danharrin, this should just be fine, right?

Reply:
Yeah, personally I like the \ though I think

Reply:
I understand, but it doesn't really make a difference as the Blade views aren't namespaces anyway. Will resolve this for now.